code/datums/supplypacks/munitions.dm

```diff
@@ -168,6 +166,16 @@
 	containername = "Magnetic weapon crate"
 	access = access_armory
 	
+/datum/supply_pack/munitions/usmc
+	name = "Weapons - M41A Pulse Rifle"
+	contains = list(/obj/item/gun/projectile/caseless/usmc,
+		/obj/item/ammo_magazine/m10x24mm/small = 2
+		)
+	cost = 200
```

Comment:
This is massively overpriced for a single gun, even if it's a assault rifle

Reply:
The M41A is intended to be prohibitively expensive. This is to balance it against the H90k as the M41A has mags that hold 96 rounds and does more damage the H90k. This is high tier Milsurp weapon that probably is still used by some professional armies in universe and the price reflects that.

That said I have knocked 50 points of the price per your recommendation. I hope though you can understand the reason for having the price so high.

Reply:
Yeah, I could understand high price point, but for 200 points as it was, I could just hand you 4 military spec Z8 Bulldogs and 2 burst laser rifles and be spending 10 points more. Though looking at cargo prices across categories is dumb, 150 points is the same as a SM crystal.

Reply:
Yeah Cargo probably needs a massive re-balancing in the future but for now. I am keeping it at 150 since everything seems to be stupid cheap anyway. I would need to consult cargo players to actually know what would be reasonable new prices for everything but I am keeping things where they are for now, save the cargo update for the future.